rage/i18n/en-US/rage.ftl

```diff
@@ -194,25 +223,287 @@ err-mnt-unknown-type = Unknown filesystem type "{$fs_type}"
+    {identities} specified with {-flag-identity} are used.
+
+    ASCII armoring is transparently detected and decoded.
+
+man-rage-flag-identity-decrypt =
+    Decrypt using the {identities} at {identity}.
+
+    {identity} may be one of the following:
+
+    a. A file listing {identities} one per line. Empty lines and lines starting
+    with "#" are ignored as comments.
+
+    b. A passphrase encrypted age file, containing {identities} one per
+    line like above. The passphrase is requested interactively. Note that
+    passphrase-protected identity files are not necessary for most use cases,
+    where access to the encrypted identity file implies access to the whole
+    system.
+
+    c. An SSH private key file, in PKCS#1, PKCS#8, or OpenSSH format.
+    If the private key is password-protected, the password is requested
+    interactively only if the SSH identity matches the file. See the
+    {man-rage-ssh-keys-heading} section for more information, including
+    supported key types.
+
+    d. {-stdin}, causing one of the options above to be read from standard input.
+    In this case, the {input} argument must be specified.
+
+    This option can be repeated. Identities are tried in the order in which are
+    provided, and the first one matching one of the file's recipients is used.
+    Unused identities are ignored, but it is an error if the {input} file is
+    passphrase-encrypted and {-flag-identity} is specified.
+
+man-rage-flag-plugin-decrypt =
+    Decrypt using the data-less plugin {plugin-name}.
+
+    This is equivalent to using {-flag-identity} with a file that contains a
+    single plugin {identity} that encodes no plugin-specific data.
+
+man-rage-recipients-and-identities-heading = RECIPIENTS AND IDENTITIES
+man-rage-recipients-and-identities =
+    {recipients} are public values, like a public key, that a file can be encrypted
+    to. {identities} are private values, like a private key, that allow decrypting
+    a file encrypted to the corresponding {recipient}.
+
+man-rage-native-x25519-keys-heading = Native X25519 keys
+man-rage-native-x25519-keys =
+    Native {-age} key pairs are generated with {-rage-keygen}(1), and provide small
+    encodings and strong encryption based on X25519. They are the recommended
+    recipient type for most applications.
+
+    A {recipient} encoding begins with "{-recipient-prefix}" and looks like the
+    following:
+
+    {"    "}{$example_age_recipient}
+
+    An {identity} encoding begins with "{-identity-prefix}" and looks like the
+    following:
+
+    {"    "}{$example_age_identity}
+
+    An encrypted file can't be linked to the native recipient it's encrypted to
+    without access to the corresponding identity.
+
+man-rage-ssh-keys-heading = SSH keys
+man-rage-ssh-keys =
+    As a convenience feature, {-rage} also supports encrypting to RSA or Ed25519
+    {-ssh} keys. RSA keys must be at least 2048 bits. This feature employs more
```

Comment:
I realised while adding this manpage that I never implemented the 2048-bit lower limit on `ssh-rsa` keys that was added to `age` in https://github.com/FiloSottile/age/issues/266. I'll open a separate PR to implement this.